### ISSUE TITLE
Understand variables like n.serviceWorker

### ISSUE THREAD

**Issue by laukstein** [amilajack/eslint-plugin-compat#102]
When code has `navigator.serviceWorker` it will return error (is Ok)

```
 22:  navigator.serviceWorker
                ^^^^^^^^^^^^^ `ServiceWorker` is not supported in IE 11, Edge 15 and Safari 8 😢
```

while when used variables like

```
var n = navigator;
n.serviceWorker.register(...);

it doesn't return error while is expected to have (not Ok).

**Comment by stereobooster** [comment]
And other way around, it complains about which detects support

```js
if ("serviceWorker" in navigator) {
  navigator.serviceWorker...
}
```
